boot/zephyr/Kconfig

```diff
@@ -234,8 +234,25 @@ config BOOT_DIRECT_XIP
 	  images must be linked to be executed from the given image slot. Using this
 	  mode results in a simpler code path and smaller code size.
 
+config BOOT_RAM_LOAD
+	bool "RAM load"
+	help
+	  If y, mcuboot selects the newest valid image based on the image version
+	  numbers, thereafter the selected image is copied to RAM and executed from
+	  there. For this reason, the image has to be linked to be executed from RAM.
+	  The address that the image is copied to is specified using the load-addr
+	  argument to the imgtool.py script which writes it to the image header.
+
 endchoice
 
+if BOOT_RAM_LOAD
+config BOOT_IMAGE_EXECUTABLE_RAM_START
```

Comment:
values for `IMAGE_EXECUTABLE_RAM_START` and `IMAGE_EXECUTABLE_RAM_SIZE` macros should have value source in DTS, similar as fixed-partitions.